File: ympa_fsharp_functions_server/src/model/AddOffersToArchiveDTO.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.AddOffersToArchiveErrorDTO

module AddOffersToArchiveDTO =

  //#region AddOffersToArchiveDTO

  [<CLIMutable>]
  type AddOffersToArchiveDTO = {
    [<JsonProperty(PropertyName = "notArchivedOffers")>]
    NotArchivedOffers : AddOffersToArchiveErrorDTO[];
  }

  //#endregion

File: src/main/scala/org/openapitools/server/model/AddOffersToArchiveDTO.scala
package org.openapitools.server.model


/**
 * Товары, которые не удалось поместить в архив.
 *
 * @param notArchivedOffers Список товаров, которые не удалось поместить в архив. for example: ''null''
*/
final case class AddOffersToArchiveDTO (
  notArchivedOffers: Option[Seq[AddOffersToArchiveErrorDTO]] = None
)

